superbit_lensing/metacalibration/ngmix_fit_superbit3.py

```diff
@@ -36,7 +40,7 @@
 parser.add_argument('--end', type=int, default=None,
                     help='Ending index for MEDS processing')
 parser.add_argument('--plot', type=str, default=True,
-                    help='Make diagnoistic plots')
+                    help='Make diagnstic plots')
```

Comment:
We both spelled it wrong lol

Reply:
We are not smarter than a fifth grader.

Reply:
Resolution: @sweverett + @mcclearyj combined brains can hopefully spell diagnostic